src/runner/driver-test-runner.js

```diff
@@ -347,8 +347,14 @@ export class DriverTestRunner {
    */
   async _collectSpeech(debounceDelay, asyncOperation) {
     let spoken = [];
+    const isJAWS = (await this.collectedCapabilities).atName == 'JAWS';
+
     const speechJob = startJob(async signal => {
-      for await (const speech of signal.cancelable(this.atDriver.speeches())) {
+      for await (let speech of signal.cancelable(this.atDriver.speeches())) {
+        if (isJAWS) {
+          // temporary workaround to double-escaped string literals.
```

Comment:
I want to keep this as visible as possible--could you write an issue and reference it here?

Reply:
#90

Reply:
Thanks!